src/routes/v2/version_creation.rs

```diff
@@ -70,6 +71,10 @@ pub struct InitialVersionData {
     pub status: VersionStatus,
     #[serde(default = "HashMap::new")]
     pub file_types: HashMap<String, Option<FileType>>,
+    // Associations to uploaded images in changelog
+    #[validate(length(max = 10))]
+    #[serde(default)]
+    pub uploaded_images: Vec<ImageId>,
```

Comment:
don't think this needs to be here anyways. users can just upload images afterwards

Reply:
resolving with discord convo